src/utils/device.js

```diff
@@ -4,13 +4,17 @@ var vrDisplay;
 if (navigator.xr) {
   navigator.xr.requestDevice().then(function (device) {
     device.supportsSession({immersive: true, exclusive: true}).then(function () {
+      var sceneEl = document.querySelector('a-scene');
       vrDisplay = device;
+      if (sceneEl) { sceneEl.emit('headsetconnected', {vrDisplay: vrDisplay}); }
```

Comment:
maybe just `display`

Reply:
It's used similarly to `checkHeadsetConnected` from utils